website/project/model.py

```diff
@@ -852,6 +852,10 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin, Commentable):
     # TODO: Add validator
     comment_level = fields.StringField(default='public')
 
+    # Project Mailing
+    mailing_enabled = fields.BooleanField(default=True, index=True)
+    mailing_updated = fields.BooleanField(default=False, index=True)
```

Comment:
Please add a comment describing what each field is, and the difference.

(especially since there are other kinds of emails than mailing lists- the node class has too many unexplained fields as is)